gen/definitions/classes.yaml

```diff
@@ -94,6 +94,21 @@ tagAnnotation:
     - "uni/tn-{name}/ctx-{name}/annotationKey-[{key}]"
     - "Too many DN formats to display, see model documentation for all possible parents of [tagAnnotation](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagAnnotation/overview)."
 
+tagTag:
+  resource_notes:
+    - "This resource should not be used in combination with the `tags` nested attributes of other resources for the same object. Doing so will result in unexpected behaviour."
+  example_classes:
+    - "fvAEPg"
+    - "fvEpIpTag"
```

Comment:
Do we want it to be mapped to another class called Tag? I would pick another second example.

Reply:
I picked this one because of the mention here: https://github.com/CiscoDevNet/terraform-provider-aci/issues/1103#issuecomment-1759326380

Can do any class you want, just let me know which class you prefer that makes most sense.

Reply:
fine. I was just afraid people would be confused with both being called Tag